### DISCUSSION TITLE
Data Persistence in the connector

### DISCUSSION THREAD

**Discussion by CTW01435** [International-Data-Spaces-Association/DataspaceConnector#71] [Q&A]
Hello, we are deploying one connector in our local environment. After creating and linking everything for one resource and adding some data to the resource we are unable to find where this data is persisted. We are using a PostrgreSQL database for the connector. Where is the actual local data for the created artifact stored? Thanks!

**Comment by juliapampus** [comment]
What version are you using? v4 or v5?

**Reply by CTW01435** [reply]
Hi,

using version 5.2.0

**Reply by jimazikerlan** [reply]
Did you manage to solve the data persistence in the connector?
